jsonschema-core.xml

```diff
@@ -724,34 +725,39 @@ Link: <http://example.com/my-hyper-schema#>; rel="describedby"
                     <artwork>
 <![CDATA[
 Content-Type: application/json;
-          profile="http://example.com/my-hyper-schema#"
+          schema="http://example.com/my-hyper-schema#"
 ]]>
                     </artwork>
                 </figure>
 
                 <t>
-                    Multiple profiles are whitespace separated:
+                    Multiple schemas are whitespace separated:
                 </t>
 
                 <figure>
                     <artwork>
 <![CDATA[
 Content-Type: application/json;
-          profile="http://example.com/alice http://example.com/bob"
+          schema="http://example.com/alice http://example.com/bob"
 ]]>
                     </artwork>
                 </figure>
 
                 <t>
-                    HTTP can also send the "profile" in a Link, though this may impact media-type
+                    <cref>
+                        This paragraph assumes that we can register a "schema" link relation.
+                        Should we instead specify something like "tag:json-schema.org,2017:schema"
+                        for now?
```

Comment:
Or a URL like `http://json-schema.org/<something here>` ? Like JSON-LD does with `http://www.w3.org/ns/json-ld#context`.

Reply:
I really like using non-dereferenceable URI schemes like `tag:` when there is nothing to dereference.  While it is certainly valid to use `http://` or `https://` for such a purpose, it is counter-intuitive and leads to a.) people blindly dumping it in a browser and being annoyed that it doesn't work and b.) awkward paragraphs in specifications about how even though the URI looks dereferenceable, implementations MUST NOT dereference it.

Just make it non-dereferenceable.  It solves all the UX problems of apparently-dereferenceable-URIs-that-are-not-actually-dereferenceable.